sloppy-deployment/entrypoint.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash -eu
 
+# Name of the Sloppy YAML configuration file
 SLOPPY_OUTPUT_FILE="sloppy.yml"
+# Name of the directory (which should be host volume) to where a copy of
+# $SLOPPY_OUTPUT_FILE is saved.
+SLOPPY_SAFE_OUTPUT_DIR="${SLOPPY_SAVE_OUTPUT:-}"
```

Comment:
Why a new variable `SLOPPY_SAFE_OUTPUT_DIR`? Why not use the already defined `SLOPPY_SAVE_OUTPUT`?

Reply:
Damn! Now I see the typer...  Fixed.